src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -108,7 +126,7 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
     }
 
     private CallbackResponse validateCorrespondenceApplicantAddress(CallbackParams callbackParams) {
-        if (callbackParams.getRequest().getEventId().equals("DEFENDANT_RESPONSE_SPEC")) {
+        if (callbackParams.getRequest().getEventId().equals(ResponseJourneyConstantsLRSpec.DEFENDANT_RESPONSE_SPEC)) {
```

Comment:
Could we do this check like this
`ResponseJourneyConstantsLRSpec.DEFENDANT_RESPONSE_SPEC.equals(callbackParams.getRequest().getEventId())`

This will cover for nullpointerexception